### ISSUE TITLE
Vagueness of a lua track "resistor-color-duo" exercise.

### ISSUE THREAD

**Issue by PowerBossNautilus** [exercism/exercism.io#6745]
The [lua resistor-color-duo exercise](https://exercism.org/tracks/lua/exercises/resistor-color-duo) reads: "The program will take color names as input and output a two digit number, even if the input is more than two colors!"

It's not clear from the description what the input type will be. It might be a table or a hyphen delimited string (The exercise suggests hyphens by using the sentence "From the example above: brown-green should return 15, and brown-green-violet should return 15 too, ignoring the third color.)". Turns out it's not true. It's a list-like table of strings instead. I learned that by looking up a test case so maybe it'd be worth noting what the input should look like in the exercise description.

I submitted a [solution](https://exercism.org/tracks/lua/exercises/resistor-color-duo/solutions/powerbossnautilus) and it failed. I had no idea why and I ended up looking up the test case again which made it clear the function I define needs to be in a table. I created my second iteration and it worked but I'm puzzled why is that needed? I would have never figured it out without reading the tests. Is storing functions in a table a well known lua convention that is obvious to lua users or am I missing something else?

To sum up:
1. Unless there's a good reason for storing the exercism user defined function in a table, don't make that a requirement. If it is needed, add this requirement in the exercise description.
2. Add the expected input type in the description.
